app/welcome/route.js

```diff
@@ -1,5 +1,11 @@
 import Ember from 'ember';
 
+export function resetDBData(model){
+  Ember.set(model, 'dbType', null);
+  Ember.set(model, 'initialDiskSize', 10);
```

Comment:
`Ember.set` can be called an Ember object, created with `Ember.Object.create`:

```
var obj = Ember.Object.create();
Ember.set(obj, 'prop', val);
```

Or with a POJO:

```
var obj = {};
Ember.set(obj, 'prop', val);
```

The POJO would not have `obj.set`, since it is just a POJO. The `set` is required since we are observing these properties elsewhere, like in the templates. This is a pretty bare data payload or configuration object, a data-bag, and I didn't feel it was appropriate to use an Ember Object (which is preferred for class/instance usage).

The POJO model for `welcome` is only used in the routing. It is not a `database` model instance.